src/app/core/services/xlsx-export.service.ts

```diff
@@ -0,0 +1,122 @@
+import { Injectable } from '@angular/core';
+import * as fileSaver from 'file-saver';
+import * as moment from 'moment';
```

Comment:
Do you know if this imports all the locale files from moment as well? I remember a discussion that this could result a bigger than necessary bundlesize to load initially, but not really sure if this still applies here.

Reply:
It seems that locale files are not imported.
https://stackoverflow.com/questions/52558679/momentjs-having-to-import-each-moment-locale-for-all-possible-languages